eclipse/plugins/org.switchyard.tools.ui/src/org/switchyard/tools/ui/Activator.java

```diff
@@ -43,6 +48,10 @@ public Activator() {
     public void start(BundleContext context) throws Exception {
         super.start(context);
         plugin = this;
+        
+        // log a usage event upon bundle startup
+        _eventType = new UsageEventType(this, PLUGIN_ID + ";startup"); //$NON-NLS-1$
```

Comment:
And please don't use new UsageEventType(Activator, String) if your bundle ID is not something like "org.jboss.tools.*".
It will use the 4th segment of the ID as a category name. Which is "ui" in this case.
Use UsageEventType(String componentName, // "switchyard"
			UsageEventType.getVersion(Activator),
			String categoryName, // "switchyard"
			String actionName) // "startup"
instead.